.gitlab-ci.yml

```diff
@@ -259,6 +259,7 @@ build_live:
     - in-isolation deploy_site
     - in-isolation create_artifact
     - in-isolation create_artifact_ignored
+    - in-isolation create_artifact_cached
     - remove_static_from_repo
```

Comment:
im sure we can make this better by have one `create_artifacts` function exposed that encompasses all of these.  will create a card for that